.github/workflows/asf-updates.yml

```diff
@@ -78,18 +78,13 @@ jobs:
           bin/release-helper.sh set-dep-ver boto3 "==$BOTOCORE_VERSION"
           bin/release-helper.sh set-dep-ver boto3-stubs "==$BOTOCORE_VERSION"
 
-          # aws-cli is two minor versions behind botocore
-          AWSCLI_VERSION=$(python -c 'import botocore; from packaging.version import Version; version = Version(botocore.__version__); print(f"{version.major}.{version.minor - 2}.{version.micro}")')
-          echo "Pinning aws-cli to version $AWSCLI_VERSION"
-          bin/release-helper.sh set-dep-ver awscli "==$AWSCLI_VERSION"
-
```

Comment:
We could theoretically treat the awscli separately, instead of basing its version on botocore.

So upgrade installing it in L40 alongside botocore. (Assuming that they won't release incompatible versions alongside each other)

And then we can pull out the selected version for awscli like we do for botocore. 

Don't know if it is worth the overhead however, as it probably won't cause any major problems being upgraded later on with all the rest.

Reply:
`awscli` is updated together with `botocore` because it has an explicit pin on a single version of `botocore`. It always has to be updated directly with `botocore`, which is why it's done here.